integration_tests/const_03.py

```diff
@@ -2,7 +2,7 @@
 
 CONST_1: Const[f64] = 32.0
 CONST_2: Const[f64] = CONST_1 * 2.0
-CONST_3: Const[i64] = CONST_1 + CONST_2
+CONST_3: Const[i64] = i64(CONST_1 + CONST_2)
```

Comment:
This shouldn't be needed I think.

Reply:
Ah I didn't notice the `i64`. Yes, it is needed, as LPython helpfully pointed out:
```console
$ lpython const_03.py 
semantic error: Type mismatch in annotation-assignment, the types must be compatible
 --> const_03.py:5:1
  |
5 | CONST_3: Const[i64] = CONST_1 + CONST_2
  | ^^^^^^^               ^^^^^^^^^^^^^^^^^ type mismatch ('i64' and 'f64')


Note: if any of the above error or warning messages are not clear or are lacking
context please report it to us (we consider that a bug that needs to be fixed).
```